runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -142,16 +153,24 @@
 		progressWrapper.add(progressBar, BorderLayout.NORTH);
 
 		container.add(headerPanel, BorderLayout.NORTH);
-		container.add(progressWrapper, BorderLayout.SOUTH);
+		container.add(progressWrapper, BorderLayout.CENTER);
 
 		container.setComponentPopupMenu(popupMenu);
 		progressBar.setComponentPopupMenu(popupMenu);
 
 		add(container, BorderLayout.NORTH);
 	}
 
+	public static String htmlLabel(String key, int value)
+	{
+		String valueStr = StackFormatter.quantityToRSDecimalStack(value);
+
+		return "<html><body style = 'color:" + SwingUtil.toHexColor(ColorScheme.LIGHT_GRAY_COLOR) + "'>" + key + "<span style = 'color:white'>" + valueStr + "</span></body></html>";
```

Comment:
Really long line, can be broken up or use string format please?

Reply:
Not my code just seems that way as I changed the formatting of the file to adhere to code conventions more properly as it didn't really before. I'm down to fix that however.